wallet/seed.go

```diff
@@ -128,14 +128,16 @@ outer:
 	for _, parent := range toSign {
 		for sigIndex, sig := range txn.Signatures {
 			if sig.ParentID == parent {
-				if addr, ok := sigAddr(parent); !ok {
+				addr, ok := sigAddr(parent)
+				if !ok {
 					return fmt.Errorf("ID %v not present in transaction", parent)
-				} else if index, ok := sav.addrs[addr]; !ok {
+				}
+				index, ok := sav.addrs[addr]
+				if !ok {
 					return fmt.Errorf("missing key for ID %v", parent)
-				} else {
-					SignTransaction(cs, txn, sigIndex, sav.seed.PrivateKey(index))
-					continue outer
 				}
+				SignTransaction(cs, txn, sigIndex, sav.seed.PrivateKey(index))
+				continue outer
```

Comment:
I understand the rationale behind this check, but I still resent it. It directly conflicts with `if`-scoped declarations, which imo are often preferable

Reply:
I didn't like this either.  Should we just disable the check here?

Reply:
can we disable it universally?

Reply:
Yeah in .golangci.yml -  I'll do it.